content/systems/filecoin_blockchain/struct/block/_index.md

```diff
@@ -2,25 +2,109 @@
 title: Block
 weight: 1
 dashboardWeight: 1.5
-dashboardState: incorrect
+dashboardState: reliable
 dashboardAudit: missing
 dashboardTests: 0
 ---
 
 # Block
 
-A block header contains information relevant to a particular point in time over which the network may achieve consensus.
+The Block is the main unit of the Filecoin blockchain.
 
-> **Note:** A block is functionally the same as a block header in the Filecoin protocol. While a block header contains Merkle links to the full system state, messages, and message receipts, a block can be thought of as the full set of this information (not just the Merkle roots, but rather the full data of the state tree, message tree, receipts tree, etc.). Because a full block is quite large, our chain consists of block headers rather than full blocks. We often use the terms `block` and `block header` interchangeably.
+The Block structure in the Filecoin blockchain is composed of: i) the Block Header, ii) the list of messages inside the block, and iii) the signed messages. This is represented inside the `FullBlock` abstraction. The messages indicate the required set of changes to apply in order to arrive at a deterministic state of the chain.
```

Comment:
BlsMessages and SecpkMessages are both signed, just differently. The single aggregate signature for the BLS  messages is in the block header. They are two distinct lists, not parallel lists of different representations of the same logical messages.